### PR TITLE
removed one email for test

### PR DESCRIPTION
**Pull request by aalikaram**
This PR implements the following **changes:**

* insert your summary of your request here, add more bullets as needed

**URL / Link to page**

### CODE REVIEW

File: content/authors/lt-col-theodore-j-erickson/_index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ last_name: "Erickson"
 pronoun: ""
 
 # Email — If you include an email address, it will be displayed on your profile page
-email: theodore.erickson@us.af.mil
```
Comment (mejiaj) [inline content/authors/lt-col-theodore-j-erickson/_index.md:15]: The spreadsheet lists this as `Federal` [here](https://docs.google.com/spreadsheets/d/1NCE1GqifPoY6t-xZDej8MPESW010lie4gTQGzLk8K6I/edit#gid=0)

File: content/authors/courtney-greenley/_index.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ pronoun: ""
 slug: courtney-greenley
 
 # if you include an email address, it will be displayed on your profile page
-email: "Courtney.L.Greenley@usace.army.mil"
```
Comment (mejiaj) [inline content/authors/courtney-greenley/_index.md:20]: Federal.

File: content/authors/albinko-hasic/_index.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ last_name: "Hasic"
 pronoun: ""
 
 # Email — If you include an email address, it will be displayed on your profile page
-email: Albinko.Hasic@usace.army.mil
```
Comment (mejiaj) [inline content/authors/albinko-hasic/_index.md:15]: Federal.

**Review by mejiaj — CHANGES_REQUESTED** [review]
Ali, please check the [ticket](https://trello.com/c/dnFbdTXt/592-clean-authors-data#comment-62150b2a9faf053e75e275e5) we're only supposed to remove non-federal emails.

**Review by mejiaj — APPROVED** [review]
LGTM, thanks Ali!

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**